bson/json_util.py

```diff
@@ -235,6 +235,8 @@ class JSONMode:
 else:
     _BASE_CLASS = CodecOptions
 
+INT32_MAX = 2**31
```

Comment:
You probably don't want people using this in their own code, so prepend it with an underscore.